server/__tests__/api.tests.js

```diff
@@ -0,0 +1,69 @@
+const request = require('supertest');
+const { PrismaClient } = require('@prisma/client');
+
+const prisma = new PrismaClient({
```

Comment:
I am not an expert on Prisma, but I don't think this is the right way to set up mock testing with Primsa. This inserts into the database which might become really slow for example if different pipelines in a CI/CD are running this.

Being looking at this [documentation](https://www.prisma.io/docs/guides/testing/unit-testing) where Prima client is mocked and tested.